### DISCUSSION TITLE
Steady-state solver

### DISCUSSION THREAD

**Discussion by TRPrasanna** [loganoz/horses3d#198] [Q&A] [answered]
All the solvers in HORSES3D seem to be solving the time-dependent governing equations. I may be wrong. Is there a recommended way of obtaining steady-state solutions (ideally something other than iterating for a large number of time-steps) ?

**Comment by loganoz** [comment]
We have implicit and multigrid solvers implemented. Take a look to chapter 7 and chapter 9 in the manual. If you are running parallel simulations with MPI, I would recommend trying FAS with RK3 and local time step to start. Implicit solver is trickier as it depends on several external libraries. There are some examples in the test cases folder.

**Reply by TRPrasanna** [reply]
When using FAS for steady-state simulations, the program outputs some value for time. May I know what this corresponds to? Is it some sort of pseudo time?

Also, by RK3 and local time-step, did you mean that I should use the time-dependent simulation with pseudo-time stepping?

**Reply by loganoz** [reply, marked as answer]
Hi @TRPrasanna, 

Yes. As you are using RK3 as MG smoother the output is the time used by the RK3 (which is not physical). 
I attach a help document a developer generated on the use of FAS for steady simulations. 
Take a look and let me know if it helps. 

[FAS - short guide.pdf](https://github.com/loganoz/horses3d/files/13827931/FAS.-.short.guide.pdf)
